src/components/Editor/MediaUploader/VideoEmbedForm.jsx

```diff
@@ -0,0 +1,69 @@
+import { Button } from "neetoui";
+import { Form, Input } from "neetoui/formik";
+import { useTranslation } from "react-i18next";
+
+import { VIDEO_EMBED_FORM_VALIDATION_SCHEMA } from "./constants";
+
+import { validateUrl } from "../CustomExtensions/Embeds/utils";
+
+const VideoEmbedForm = ({ onSubmit, onCancel }) => {
+  const { t } = useTranslation();
+
+  const handleSubmit = ({ videoUrl }) => {
+    const validatedUrl = validateUrl(videoUrl);
+    if (validatedUrl) {
+      onSubmit(videoUrl);
+    }
+  };
+
+  return (
+    <div className="ne-video-embed-form">
+      <Form
+        formikProps={{
+          initialValues: { videoUrl: "" },
+          onSubmit: handleSubmit,
```

Comment:
Since we validate the URL using Yup, we need not validate it again in `handleSubmit` as well. Hence, directly use the `onSubmit` function here and drop the `handleSubmit` function definition.

```suggestion
        onSubmit: ({ videoUrl }) => onSubmit(videoUrl),
```